models/cfscript/Functions.cfc

```diff
@@ -69,6 +74,11 @@ component {
                 !attributesTxt.find(chr(10))
             ) {
                 formattedText = formattedText & ' ';
+            } else if (
+                settings['#rootSettingKey#.group_to_block_spacing'] == 'expanded' &&
+                !attributesTxt.find(chr(10))
+            ) {
+                formattedText = formattedText.replace('(', ' (') & ' ';
```

Comment:
Shouldn't this else block be removed now?

Reply:
Removed old "expanced" if block as it is no longer utilized.